plugins/extraction/extract_env.ml

```diff
@@ -351,17 +351,17 @@ and extract_mexpression env mp = function
          extract_mbody_spec env mp1 mtb,
          extract_mexpression env' mp me)
 
-and extract_msignature env mp reso ~all = function
+and extract_msignature env mp reso sealed ~all = function
```

Comment:
Using a named parameter `~sealed` may be a tad cleaner IMHO.

Reply:
You mean because `all` is already named? Ok, done.